src/page/root.rs

```diff
@@ -12,7 +12,13 @@ async fn get_top_post_id() -> Result<Option<(Principal, u64)>, ServerFnError> {
     let post_cache = canisters.post_cache();
 
     let top_items = match post_cache
-        .get_top_posts_aggregated_from_canisters_on_this_network_for_hot_or_not_feed(0, 1)
+        .get_top_posts_aggregated_from_canisters_on_this_network_for_home_feed_cursor(
+            0,
+            1,
+            None,
+            None,
+            Some(post_cache::NsfwFilter::ExcludeNsfw),
```

Comment:
Are we not using filters for Home feed ?

Reply:
The first post that's fetched does not use filters for a fast TTL, all subsequent posts follow the filter
see https://github.com/go-bazzinga/hot-or-not-web-leptos-ssr/blob/main/src/page/post_view/video_iter.rs#L144